tests/acceptance/features/webUISharingExternal/federationSharing.feature

```diff
@@ -359,3 +359,10 @@ Feature: Federation Sharing - sharing with users on other cloud storages
     And the user opens the share dialog for file "textfile.txt" using the webUI
     Then remote user "user1" should be listed as "Editor" via "simple-folder" in the collaborators list on the webUI
     And user "User Two" should be listed as "Editor" via "sub-folder" in the collaborators list on the webUI
+
+    Scenario: shares folder appears only after reloading the page
+      When user "user1" from remote server shares "simple-folder" with user "user1" from local server
```

Comment:
Maybe you can do this in `Given`. Anyway, it looks like the new step definition does the share using API.